Changes to the file build.gradle and application.properties

pszczepcio opened the pull request.

pszczepcio committed:
Changes to the file build.gradle and application.properties

klikuskodilla commented:
Zwróć uwagę że CircleCI rzuca błędem podczas uruchamiania testów jednostkowych:
2019-05-01 19:58:31.688 ERROR 439 --- [       Thread-6] o.h.engine.jdbc.spi.SqlExceptionHelper   : Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]

klikuskodilla reviewed (changes requested):
Zwróć uwagę że CircleCI rzuca błędem podczas uruchamiania testów jednostkowych:
2019-05-01 19:58:31.688 ERROR 439 --- [ Thread-6] o.h.engine.jdbc.spi.SqlExceptionHelper : Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]

pszczepcio commented:
W którym miejscu jest jest ten błąd CircleCI. Nie mogę tego namierzyć i teraz nie wiem jak sprawdzić czy dalej będę miał ten błąd czy nie

pszczepcio committed:
- add to URL : DB_CLOSE_ON_EXIT=FALSE
-change @GeneratedValue(strategy = GenerationType. Auto) on @GeneratedValue(strategy = GenerationType.IDENTITY)

klikuskodilla commented:
Możesz spradzić rezultat builda konkretnego commita klikając jak na obrazku
![image](https://user-images.githubusercontent.com/33075397/57182439-59629880-6e9f-11e9-836d-3f7c965225b0.png)

Najnowszy rezultat builda będzie się wyświetlał w tym miejscu
![image](https://user-images.githubusercontent.com/33075397/57182449-7b5c1b00-6e9f-11e9-950a-ddf72b8a5578.png)
